generated/nidaqmx/_grpc_interpreter.py

```diff
@@ -3392,10 +3438,26 @@ def _assign_numpy_array(numpy_array, grpc_array):
     grpc_array_size = len(grpc_array)
     if isinstance(grpc_array, bytes):
         assert numpy_array.nbytes >= grpc_array_size
-        numpy_array.flat[:grpc_array_size] = numpy.frombuffer(grpc_array, dtype=numpy_array.dtype)
+        if numpy_array.ndim>1:
+            (number_of_channels, number_of_samples_per_channel) = numpy_array.shape
+            read_array = numpy.frombuffer(grpc_array, dtype=numpy_array.dtype).reshape((number_of_channels, samples_read))
+            for row in range(0, number_of_channels):
+                for column in range(0, number_of_samples_per_channel):
+                    if column < samples_read:
+                        numpy_array[row, column] = read_array[row, column]
+        else:
+            numpy_array.flat[:grpc_array_size] = numpy.frombuffer(grpc_array, dtype=numpy_array.dtype)
```

Comment:
why does this case not care about `samples_read`?

Reply:
I have updated the logic to include `sample_read` while converting bytes into numpy array